### PR TITLE
add the Spanish translation for the post of your first robot, part1

### PR DESCRIPTION
**Pull request by come-maiz**
Notes for reviewers:

* [English post](https://github.com/elopio/snapcraft-videos/blob/master/your_first_robot/part1/post.md)
* Please leave your review comments in English.

### CODE REVIEW

**Comment by angel-araya** [comment]
`## Step 1: Instalar Ubuntu Core`

Shouldn't the "Step N" headers be in spanish too? ("Paso 1", "Paso 2", etc)

**Comment by come-maiz** [comment]
@angelaraya they should, yes. Nice catch. I will make the change.

File: your_first_robot/part1/post-es.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+
+Como mencioné, el las hojas de trabajo de CamJam nos llevarán lejos hacia lo que necesitamos aprender para esta serie. Sin embargo, en lugar de usar Raspbian, vamos a usar Ubuntu Core. Las hojas de trabajo aún son perfectamente aplicables, y lo probaré guiándole a través de la configuración de Ubuntu Core en su Raspberry Pi, y luego seguiremos una de las hojas de trabajo de CamJam. Note que esta también es una serie de vídeos, siéntase libre de ver la versión en vídeo de este post (con subtítulos en Español):
+
+[![Vídeo: Su primer robot, parte 1: una guía introductoria a ROS y Ubuntu Core](http://img.youtube.com/vi/KidVVqbsIHI/0.jpg)](http://www.youtube.com/watch?v=KidVVqbsIHI)
+
+## Prerrequisitos
+
+Aunque esto es una introducción, quiero ser claro sobre algunas de las cosas que estoy asumiendo. Esta serie asume que usted está familiarizada un poco con Ubuntu (o alguna otra distro basada en Debian, como Raspbian), en particular, usando la línea de comandos. No espero que sea una profesional escriiendo scripts, pero debería saber cómo navegar el sistema de archivos y usar editores de terminal (vi, nano, etc.)
+
+## ¿Qué es Uuntu Core?
```
Comment (angel-araya) [inline your_first_robot/part1/post-es.md]: Uuntu > Ubuntu

File: your_first_robot/part1/post-es.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+
+Entonces, ¿cómo instala una todas las herramientas de desarrollo a las que está acostumbrada? Hay un snap especial para eso, llamado **classic**, que le da acceso a todas su herramientas conocidas (incluyendo **apt**).
+
+Instale el snap **classic** siguiendo la sección de «developing on target» (desarrollando en el mismo sistema objetivo) de la [guía de configuración de desarrollo](https://developer.ubuntu.com/core/get-started/developer-setup) (de nuevo, comente aquí si tiene preguntas). Al final de este paso, usted debe poder ejecutar **sudo classic** y obtener acceso a un shel en el que puede instalar debs, ¡el cual usaremos para hackear en nuestro robot! Luego crearemo su propio snap para controlar el robot, lo que hace super fácil.
```
Comment (angel-araya) [inline your_first_robot/part1/post-es.md]: shel > shell

File: your_first_robot/part1/post-es.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+## Step 3: Hora de CamJam
+
+Muy bien, ¡ahora estamos en el punto en el que podemos empezar a seguir las hojas de trabajo de CamJam! ¿Por qué no empezamos con la [primera](https://github.com/CamJam-EduKit/EduKit3/raw/master/CamJam%20EduKit%203%20-%20Robotics%20Worksheet%201%20-%20Introduction.pdf). Esta en realidad asume que usted está ejecutando Raspbian, lo que no estamos haciendo, entonces aunque puede leerla toda, vamos a saltar directo al paso «Identifying your Version of Raspbian» (Identificando su versión de Raspbian). No, aún no estamos usando Raspbian, pero esta sección tiene un punto interesante que quiero asegurarme de que note:
```
Comment (angel-araya) [inline your_first_robot/part1/post-es.md]: Missing closing question mark. "[primera]**_?_**"

File: your_first_robot/part1/post-es.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+Siga adelante y lea el resto de la hoja de trabajo, le será útil luego.
+
+Espero que esto le sirva como un buen inicio para usar Ubuntu Core para hackear en sus proyectos con Raspberry Pi. Como tarea para el resto de la serie, complete también las hojas de trabajo 2-4, y 7. Por supuesto puede hacer las otras también, pero nosotras no usaremos el sensor ultrasónico o el detector de línea en esta serie, entonces no tiene que. Pero, antes de hacer las otras hojas de trabajo, necesitará instalar el paquete RPi.GPIO. En su shell classic, ejecute:
+
+```
+$ sudo apt install gcc python3-dev python3-pip python3-setuptools
+$ pip3 install RPi.GPIO
+```
+
+También recuerde que necesitará usar sudo para cualquier script de python que utilice GPIO. El [siguiente post en esta serie](https://kyrofa.com/posts/your-first-robot-introduction-to-the-robot-operating-system-2-5) sera una introducción a ROS, ¿qué es?, ¿por qué es útil?, ¿por qué necesita conocerlo? y ¿cómo obtenerlo en su Raspberry Pi con Ubuntu Core.
```
Comment (angel-araya) [inline your_first_robot/part1/post-es.md]: python > Python
Missing closing question mark. "Ubuntu Core**_?_**"

**Review by angel-araya — COMMENTED** [review]
Highlighted some typos and missing question marks.

File: your_first_robot/part1/post-es.md [hunk 5]
```diff
@@ -0,0 +1,82 @@
...
+  * Esto es opcional, pero si no tiene uno, su Pi estará amarrado a su fuente de poder.
+* [Control inalámbrico](https://thepihut.com/collections/raspberry-pi-gaming/products/raspberry-pi-compatible-wireless-gamepad-controller?variant=38135423121), £14, digamos, $20
+* Total, incluyendo componentes opcionales: $95
+
+En mi serie de robótica anterior, una de las razones por las que el Turtlebot era tan caro es porque tiene una enorme cantidad de software ya escrito, como controladores ROS para movimiento, leer de sensores, etc. Este pequeño robot es diferente: necesitaremos escribir todo por nosotras mismas. Por suerte, la gente de CamJam brinda unas [hojas de trabajo](http://camjam.me/?page_id=1035) para introducirle a la plataforma y ayudarle a empezar a escribir software para controlar las ruedas, leer el sensor ultrasónico, y más. Podemos aprovecharlas para hacer funcionar el robot de forma rápida. Entonces, a través de esta serie cubriremos los siguientes temas:
+
+* [Su primer robot: introduccion al Robot Operating System [2/5]](https://kyrofa.com/posts/your-first-robot-introduction-to-the-robot-operating-system-2-5)
```
Comment (allanesqui) [inline your_first_robot/part1/post-es.md]: introduccion > introducción

File: your_first_robot/part1/post-es.md [hunk 6]
```diff
@@ -0,0 +1,82 @@
...
+  * ¿Qué es el Robot Operating System (ROS)? ¿Por qué vale la pena aprenderlo? ¿Cómo usarlo?
+* Su primer robot: el control remoto [3/5]
+  * Obteniendo datos del control inalámbrico y usándolos en ROS.
+* Su primer robot: el controlador [4/5]
+  * Aprender un poco de teoría de control, escribir el controlador para el robot.
+* Su primer robot: compartiendo con otras [5/5]
+  * ¿Cómo se arranca el sistema sin tener que entrar por SSH? ¿Cómo compartir su producto final con sus amistades?
+
+Como mencioné, el las hojas de trabajo de CamJam nos llevarán lejos hacia lo que necesitamos aprender para esta serie. Sin embargo, en lugar de usar Raspbian, vamos a usar Ubuntu Core. Las hojas de trabajo aún son perfectamente aplicables, y lo probaré guiándole a través de la configuración de Ubuntu Core en su Raspberry Pi, y luego seguiremos una de las hojas de trabajo de CamJam. Note que esta también es una serie de vídeos, siéntase libre de ver la versión en vídeo de este post (con subtítulos en Español):
```
Comment (allanesqui) [inline your_first_robot/part1/post-es.md]: need to delete 3rd word "el", el las hojas > las hojas

File: your_first_robot/part1/post-es.md [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+
+[![Vídeo: Su primer robot, parte 1: una guía introductoria a ROS y Ubuntu Core](http://img.youtube.com/vi/KidVVqbsIHI/0.jpg)](http://www.youtube.com/watch?v=KidVVqbsIHI)
+
+## Prerrequisitos
+
+Aunque esto es una introducción, quiero ser claro sobre algunas de las cosas que estoy asumiendo. Esta serie asume que usted está familiarizada un poco con Ubuntu (o alguna otra distro basada en Debian, como Raspbian), en particular, usando la línea de comandos. No espero que sea una profesional escriiendo scripts, pero debería saber cómo navegar el sistema de archivos y usar editores de terminal (vi, nano, etc.)
```
Comment (allanesqui) [inline your_first_robot/part1/post-es.md]: escriiendo > escribiendo

**Review by allanesqui — COMMENTED** [review]
Submitting spelling errors

File: your_first_robot/part1/post-es.md [hunk 8]
```diff
@@ -0,0 +1,82 @@
...
+
+## Step 1: Instalar Ubuntu Core
+
+Esto ya está bien documentado, pero quiero agregar unas cuantas notas antes de que siga los pasos. Antes que todo, se requiren un teclado y un monitor, pero solo para el primer arranque. A diferencia de Raspbian o Ubuntu MATE, no hay nombre de usuario ni contraseña predefinidas, o un proceso de instalación en el que se cree un usuario. Ni siquiera utiliza un ratón-- esto es más como Ubuntu Server. De hecho, no hay un inicio de sesión local: todo se hace a través de SSH (una sesión remota a través de la red), con claves criptográficas en lugar de contraseñas. Esto es parte de la filosofía de «seguridad predeterminada» de Ubuntu Core. Hay muchos ejemplos de credenciales predefinidas siendo abusadas (vea la [red de bots Mirai](https://en.wikipedia.org/wiki/Mirai_(malware))), y como un sistema operativo enfocado en IoT, ¡Ubuntu Core no puede tener eso! Esa es la razón por la que verá que una cuenta de Ubuntu SSO y claves SSH son prerrequisitos en la guía de instalación (no se preocupe, ambas son fáciles, ¡y su robot estará super seguro!).
```
Comment (allanesqui) [inline your_first_robot/part1/post-es.md]: I would change the word "abusadas", it's the literal meaning of abused. But I think, in spanish there's another word for this, maybe a tech word for this in spanish.

File: your_first_robot/part1/post-es.md [hunk 9]
```diff
@@ -0,0 +1,82 @@
...
+
+Bueno, sigamos la [guía de instalación de Raspberry Pi 2 o 3](https://developer.ubuntu.com/core/get-started/raspberry-pi-2-3) (comente aquí si tiene alguna pregunta). Al final de este paso, usted tendrá Ubuntu Core instalado, y debería poder entrar al Pi a través de SSH.
+
+## Step 2: Preparar el ambiente de desarrollo
+
+Como un sistema operativo enfocado en IoT, Ubuntu Core usa un formato de empaquetamiento diferente al de Raspbian y Ubuntu clásico. En lugar de usar paquetes Debian (que involucran familiaridad con herramientas como **apt**), Ubuntu Core usa un formato de paquetes llamado **snaps**. Los snaps son mucho más robustos cuando se trata de actualizaciones y seguridad, ambas importantes para dispositivos IoT, incluyendo robots. No hay **apt** en este sistema. Para darle una prueba de cómo es usar los comandos «snap», asegúrese que su sistema está completamente actualizado corriendno **snap refresh**.
```
Comment (allanesqui) [inline your_first_robot/part1/post-es.md]: Maybe "snaps" in capital letter.

Comment (allanesqui) [inline your_first_robot/part1/post-es.md]: corriendno > corriendo

File: your_first_robot/part1/post-es.md (hunk 8 above)
Comment (angel-araya) [reply your_first_robot/part1/post-es.md]: @allanesqui The original text uses "abused" as well. I'd incline to leave it as is to match the text 1:1. However I agree that the wording could be improved on both languages to better reflect the meaning of this to a beginner.

File: your_first_robot/part1/post-es.md (hunk 2 above)
Comment (allanesqui) [inline your_first_robot/part1/post-es.md]: crearemo > crearemos

Comment (allanesqui) [inline your_first_robot/part1/post-es.md]: which makes sharing your project with your friends super easy:

lo que hace super fácil > lo que hace compartir su proyecto con amigos super fácil.

File: your_first_robot/part1/post-es.md [hunk 10]
```diff
@@ -0,0 +1,82 @@
...
+
+## Step 3: Hora de CamJam
+
+Muy bien, ¡ahora estamos en el punto en el que podemos empezar a seguir las hojas de trabajo de CamJam! ¿Por qué no empezamos con la [primera](https://github.com/CamJam-EduKit/EduKit3/raw/master/CamJam%20EduKit%203%20-%20Robotics%20Worksheet%201%20-%20Introduction.pdf). Esta en realidad asume que usted está ejecutando Raspbian, lo que no estamos haciendo, entonces aunque puede leerla toda, vamos a saltar directo al paso «Identifying your Version of Raspbian» (Identificando su versión de Raspbian). No, aún no estamos usando Raspbian, pero esta sección tiene un punto interesante que quiero asegurarme de que note:
+
+Ubuntu Core, al igual que Debian Wheezy, considera que el acceso a GPIO a través de mapeo de memoria es una operación privilegiada. Como resultado, cualquier código a lo largo de estas hojas de trabajo que usa GPIO necesitará que lo ejecute con **sudo**.
+
+Todo bien, empecemos. Asegúrese que está en su shell classic ejecutando **sudo classic**. vi está disponible aquí, pero puede **sudo apt install nano** si quiere. Siguiendo las hojas de trabajo, abra **1-helloworld.py** y llénelo con lo siguiente:
```
Comment (allanesqui) [inline your_first_robot/part1/post-es.md]: vi está disponible aquí > vi está disponible predeterminadamente aquí

**Review by allanesqui — COMMENTED** [review]
correcting some spelling issues and two comments to consider

File: your_first_robot/part1/post-es.md (hunk 8 above)
Comment (come-maiz) [reply your_first_robot/part1/post-es.md]: I'm happy with "abusadas". If you have another option, put it here and we can discuss it.

File: your_first_robot/part1/post-es.md (hunk 10 above)
Comment (come-maiz) [reply your_first_robot/part1/post-es.md]: We don't need to make a literal translation, and "predeterminadamente" sounded bad to me. Do you think it makes it better?

**Comment by come-maiz** [comment]
This has been amazing! I made so many mistakes, I'm so happy I got you reviewing it before we published it.
I think I've applied everything, and I replied in a few comments. Please take another look and approve the PR if it looks good to publish. 
If you disagree with some of my comments, or want to make more corrections, feel free to continue the discussion :)

**Pull request merged**